modules/nf-core/htseq/count/main.nf

```diff
@@ -0,0 +1,51 @@
+
+
+process HTSEQ_COUNT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::htseq=2.0.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/htseq:2.0.2--py310ha14a713_0':
+        'biocontainers/htseq:2.0.2--py310ha14a713_0' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    tuple val(meta1), path(gtf)
```

Comment:
```suggestion
    tuple val(meta2), path(gtf)
```
All modules use meta2 for the second meta map (just so you stay consistent with the other modules)